script.js

```diff
@@ -0,0 +1,71 @@
+if (localStorage.getItem('data') === null) {
+  localStorage.setItem('data', '[]');
+}
+
+function Book(id, title, author) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+}
```

Comment:
* The collection for storing a list of the created books is missing, please fix it as currently, books created are not stored